generate-stackbrew-library.sh

```diff
@@ -5,7 +5,7 @@ shopt -s globstar
 declare -A aliases
 aliases=(
 	[9.2-jre7]='jre7'
-	[9.3-jre8]='latest jre8'
+	[9.4-jre8]='latest jre8'
```

Comment:
@md5, we have always run point  releases in parallel for a while and expect to do so for at least a little while.   So I would expect perhaps 1 or 2 more 9.3.x releases.   However as there are no technical barriers to upgrade to 9.4 (same JVM same API), then it will only those that need to port to changed APIs (sessions mostly) that may linger on 9.3

Note that in 9.4, we have added some features that make jetty easier to configure in a Dockerfile - better support for property substitution and the ability to put the confuguration options in a file rather than on the command line.   I'll have a look next week to see if there is scope to use those in 9.4, or at least include reference to them in the documentation.